.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: node_js
 
 node_js:
-  - '4.2.1'
+  - '8'
```

Comment:
The commit message mentions the mongodb version bump, but you also bumped the node JS bump to a new major version. Might be a good idea to update the commit message to reflect your changes a little more accurate. Your future self might thank you 😉